controllers/admin/AdminProductsController.php

```diff
@@ -889,6 +889,7 @@ public function processFeatures($id_product = null)
                         }
                     }
                 }
+	            Search::indexation(false, $id_product);
```

Comment:
Could you please add a little comment to explain this line of code?

Reply:
Added additional indexation method straight after the new feature is recorded to the DB. It fixes the issue with Search::getFeatures method where it is run before new feature values are saved.

Reply:
Sorry, I mean in the code :p as a phpdoc =)